src/main/java/com/simibubi/create/foundation/blockEntity/renderer/SafeBlockEntityRenderer.java

```diff
@@ -42,6 +45,6 @@ public boolean shouldCullItem(Vec3 itemPos) {
 				itemPos.z + 0.25
 		);
 
-		return !frustum.isVisible(itemBB);
+		return !(frustum.isVisible(itemBB) || level instanceof PonderWorld);
```

Comment:
The `instanceof` check should be done before retrieving the frustum and constructing an AABB, as it will be cheaper than the allocation